float64 max_linear_velocity
float64 max_angular_velocity
---
bool success
string message